### PR TITLE
fix(listbox): card shadow clipping when inside listbox item

### LINKED ISSUE #3480: [BUG] -  Card's shadow cut off when nested inside ListBoxItem
### NextUI Version

2.4.2

### Describe the bug

The shadow of a Card component nested inside a ListBoxItem is cut off. The `overflow: hidden` property in the `truncate` class appears to be causing this issue.

QQ_1721108740651
*Notice the corner of the Card*

### Your Example Website or App

_No response_

### Steps to Reproduce the Bug or Issue

1. Create a new project
2. Add listbox and card
3. App.tsx
```tsx
import { Card, CardBody, CardHeader } from "@nextui-org/card";
import { Listbox, ListboxItem } from "@nextui-org/listbox";

function App() {
  return (
    <Listbox aria-label="test">
      <ListboxItem key="test" textValue="test">
        <Card shadow="lg">
          <CardHeader>header</CardHeader>
          <CardBody>body</CardBody>
        </Card>
      </ListboxItem>
    </Listbox>
  );
}

export default App;
```

### Expected behavior

The shadow of the Card should be fully visible and not cut off when nested inside a ListBoxItem. 

### Screenshots or Videos

image
*shadow is cut off*

image
*after change `overflow:hidden` to `overflow:visible`*

### Operating System Version

Windows

### Browser

Chrome

### PR DESCRIPTION
**Pull request by awesome-pro**
Closes #3480

## 📝 Description

Fixed the clipping of card shadow, when present inside listbox-item or menubox-item

## ⛳️ Current behavior (updates)

Screenshot 2024-07-17 at 3 01 56 AM

## 🚀 New behavior

Screenshot 2024-07-17 at 3 02 36 AM

## Summary by CodeRabbit

- **Bug Fixes**
  - Resolved an issue with shadow clipping for cards inside listbox or menubox items.
  - Improved text overflow behavior in menu items by adjusting the style attributes.

### CODE REVIEW

File: .changeset/five-teachers-refuse.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+"@nextui-org/theme": patch
+---
+
+fixed the shadow clipping of card inside listbox or menubox item
```
Comment (wingkwong) [inline .changeset/five-teachers-refuse.md:5]: link issue number

```
fixed the shadow clipping of card inside listbox or menubox item (#3480)
```

**Review by wingkwong — CHANGES_REQUESTED** [review]

File: packages/core/theme/src/components/menu.ts [hunk 2]
```diff
@@ -71,7 +71,7 @@ const menuItem = tv({
       "data-[focus-visible=true]:dark:ring-offset-background-content1",
     ],
     wrapper: "w-full flex flex-col items-start justify-center",
-    title: "flex-1 text-small font-normal truncate",
+    title: "flex-1 text-small font-normal overflow-visible",
```
Comment (wingkwong) [inline packages/core/theme/src/components/menu.ts:74]: by default `overflow` is already visible if there is no truncate.

**Review by wingkwong — CHANGES_REQUESTED** [review]

**Comment by abhinandan-verma** [comment]
@wingkwong  sir I already tried that, but it didn't work in my case. Same issue was coming 
I tried it inside listbox storybook

**Comment by abhinandan-verma** [comment]
@wingkwong  kindly review it, I have made necessary changes

**Review by wingkwong — APPROVED** [review]

**Pull request merged**